efficientdet/det_model_fn_test.py

```diff
@@ -0,0 +1,36 @@
+# Lint as: python3
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for det_model_fn."""
+import tensorflow as tf
+
+import det_model_fn
+
+
+class DetModelFnTest(tf.test.TestCase):
+
+  def test_focal_loss(self):
```

Comment:
@Ely-S I added a simple test to ensure the new focal_loss function has the same output if label_smoothing=0.  Could you also add a test to ensure the output is expected when label_smoothing != 0?